package.json

```
@@ -4,10 +4,10 @@
   "description": "OpenTelemetry is a distributed tracing and stats collection framework.",
   "scripts": {
     "precompile": "lerna run version",
-    "compile": "lerna run protos && tsc --build",
+    "compile": "lerna run protos && tsc --build tsconfig.json tsconfig.esm.json tsconfig.esnext.json",
     "prewatch": "npm run precompile",
-    "watch": "tsc --build --watch",
-    "clean": "tsc --build --clean",
+    "watch": "tsc --build --watch tsconfig.json tsconfig.esm.json tsconfig.esnext.json",
+    "clean": "tsc --build --clean tsconfig.json tsconfig.esm.json tsconfig.esnext.json",
     "postinstall": "npm run update-ts-references && npm run bootstrap",
```

looks like this has not been updated :slightly_smiling_face: 
```suggestion
    "postinstall": "npm run update-ts-configs && npm run bootstrap",
```